docs/getting_started/install_using_ddev.md

```diff
@@ -0,0 +1,397 @@
+
+```bash
+# Clone the version-controlled project and enter its local directory
+git clone <repository> my-ddev-project && cd my-ddev-project
+# Exclude the whole `.ddev/` directory from version control
+.ddev/ >> .gitignore
+# Configure the DDEV project then start it
+ddev config --project-type=php --php-version 8.1 \
+  --docroot=public \
+  --web-environment-add DATABASE_URL=mysql://db:db@db:3306/db \
+  --http-port=8080 --https-port=8443
+ddev start
+# Configure Composer authentication
+ddev composer config --global http-basic.updates.ibexa.co <installation-key> <token-password>
+# Install the dependencies packages
+ddev composer install
+# Populate the database with a clean install
+ddev php bin/console ibexa:install ibexa-<edition>
+# Add some content types using a migration file (previously created on another installation) and update the GraphQL schema
+ddev php bin/console ibexa:migrations:migrate --file=project_content_types.yaml
+ddev php bin/console ibexa:graphql:generate-schema
+# Open the project in the default brother which should display the default SiteAccess frontpage
+ddev launch
+```
+
+Notice that the example adds the whole `.ddev/` directory to `.gitignore`, but you can also version parts of it.
+Some DDEV configs can be shared among developers. For example, a common `.ddev/config.yaml` can be committed for everyone and [locally extended or overridden](https://ddev.readthedocs.io/en/latest/users/extend/customization-extendibility/#extending-configyaml-with-custom-configyaml-files).
+
+Compared to running a clean install like described in [Installation steps](#installation-steps):
+
+- Instead of creating an empty directory like in [1. Create a DDEV project directory](#1-create-a-ddev-project-directory), you can use an existing directory containing an Ibexa DXP project.
+- In [2. Configure DDEV / Configure PHP version and document root](#configure-php-version-and-document-root), don't create the Document root, remove the `--create-docroot` option.
+- Instead of `ddev composer create`, in [5. Create Ibexa DXP project](#5-create-ibexa-dxp-project) use only `ddev composer install`.
+- [Ibexa data migration](../content_management/data_migration/importing_data.md) or [`ddev import-db`](https://ddev.readthedocs.io/en/latest/users/usage/commands/#import-db) can be used to populate the database.
+
+### Hostnames and domains
+
+If the local project needs to answer to real production domains (for example, to use the existing [hostname to SiteAccess](../multisite/siteaccess/siteaccess_matching.md#maphost) or [hostname element to SiteAccess](../multisite/siteaccess/siteaccess_matching.md#hostelement) mappings), you can use [additional hostnames](https://ddev.readthedocs.io/en/latest/users/extend/additional-hostnames/).
+
+!!! warning
+
+    As this feature will modify domain resolution, the real website may be unreachable until the `hosts` file is manually cleaned.
+
```

Comment:
@DominikaK  'warning' has different styling from others: tip, caution, notice, is it by design?

Reply:
@juskora 

Thx. I meant "caution."

Reply:
```suggestion
!!! caution

    As this feature will modify domain resolution, the real website may be unreachable until the `hosts` file is manually cleaned.

```